diff -up crosstool-0.26/all.sh.sarge crosstool-0.26/all.sh
--- crosstool-0.26/all.sh.sarge	2018-10-05 06:33:40.357120197 +0200
+++ crosstool-0.26/all.sh	2018-10-05 06:33:40.360120202 +0200
@@ -46,7 +46,7 @@ fi
 # Change or override these to your taste.
 TARBALLS_DIR=${TARBALLS_DIR-$TOP_DIR/tarballs}
 RESULT_TOP=${RESULT_TOP-$TOP_DIR/result}
-PREFIX=/usr/arm/$TOOLCOMBO
+PREFIX=$DAUM_PREFIX/$TOOLCOMBO
 
 export TOOLCOMBO
 export PREFIX
diff -up crosstool-0.26/demo.sh.sarge crosstool-0.26/demo.sh
--- crosstool-0.26/demo.sh.sarge	2018-10-05 06:33:40.358120198 +0200
+++ crosstool-0.26/demo.sh	2018-10-05 06:33:40.361120204 +0200
@@ -1,6 +1,6 @@
 #!/bin/sh
 set -ex
-export TARBALLS_DIR=~/downloads
+export TARBALLS_DIR=$DAUM_SOURCE
 
 # To use all.sh, just set the eight or nine environment variables it wants, then run it.
 # Or better yet, read it, understand it, and *then* run it.
diff -up crosstool-0.26/gmc-arm9tdmi.dat.sarge crosstool-0.26/gmc-arm9tdmi.dat
--- crosstool-0.26/gmc-arm9tdmi.dat.sarge	2018-10-05 06:33:40.358120198 +0200
+++ crosstool-0.26/gmc-arm9tdmi.dat	2018-10-05 06:33:40.361120204 +0200
@@ -1,4 +1,4 @@
-KERNELCONFIG=$PWD/gmc/arm.config
+KERNELCONFIG=$PWD/arm.config
 TARGET=arm-9tdmi-linux-gnu
 GCC_EXTRA_CONFIG="--with-cpu=arm9tdmi --enable-cxx-flags=-mcpu=arm9tdmi"
 TARGET_CFLAGS="-O"
diff -up crosstool-0.26/getandpatch.sh.sarge crosstool-0.26/getandpatch.sh
--- crosstool-0.26/getandpatch.sh.sarge	2004-01-17 00:04:02.000000000 +0100
+++ crosstool-0.26/getandpatch.sh	2023-04-07 16:24:56.853497587 +0200
@@ -43,7 +43,11 @@ getUnpackAndPatch() {
 	cd $SRC_DIR
 
 	if test $ZIP_METHOD = "gz" ; then
-	    tar -xzvf $TARBALLS_DIR/$ARCHIVE_NAME || abort cannot unpack $TARBALLS_DIR/$ARCHIVE_NAME
+	    if test "$(file -ib $TARBALLS_DIR/$ARCHIVE_NAME  | cut -d, -f1)" = "application/x-tar"; then
+		tar -xvf $TARBALLS_DIR/$ARCHIVE_NAME || abort cannot unpack $TARBALLS_DIR/$ARCHIVE_NAME
+	    else
+		tar -xzvf $TARBALLS_DIR/$ARCHIVE_NAME || abort cannot unpack $TARBALLS_DIR/$ARCHIVE_NAME
+	    fi
 	elif test $ZIP_METHOD = "bz2"; then
 	    tar -xjvf $TARBALLS_DIR/$ARCHIVE_NAME || abort cannot unpack $TARBALLS_DIR/$ARCHIVE_NAME
 	else 
